### DISCUSSION TITLE
Add feature to expand list of columns of a query from symbol * after pressing Tab

### DISCUSSION THREAD

**Discussion by ask9** [dbeaver/dbeaver#34622] [Q&A] [answered]
Add feature to expand list of columns of a query from symbol * after pressing Tab when cursor is set after the symbol *.
See this gif-file for an example:

![0000002581](https://github.com/dbeaver/dbeaver/assets/33929221/f121015a-e44b-4627-b482-e2c82928d5c9)

**Comment by E1izabeth** [comment, marked as answer]
You can do the same in DBeaver with `Ctrl+Space`

https://github.com/dbeaver/dbeaver/assets/28875055/f7a49378-ba0e-49f0-95c9-e7d571349918

**Reply by nxlcis** [reply]
Hi E1izabeth, that is not my experience. When I hit Ctrl+Space after the *, I only get the popup listing all the columns. I am on Windows 11, and am using a vanilla DBeaver Community Edition 24.3.4. Any ideas why you get the * to expand into all the columns and I only see the popup?
![image](https://github.com/user-attachments/assets/c4187507-c8b4-4945-a798-bee2d1dd8780)

**Reply by E1izabeth** [reply]
Here is a bug about that #37042
As a temporary workaround, you can write * without alias, it works this way

**Reply by nxlcis** [reply]
Thank you for your quick response! I was able to make it work.
